pkg/local_object_storage/engine/control.go

```diff
@@ -21,7 +21,9 @@ func (e *StorageEngine) open() error {
 
 	for id, sh := range e.shards {
 		if err := sh.Open(); err != nil {
-			return fmt.Errorf("open shard %s: %w", id, err)
+			if !e.cfg.isIgnoreMissingShards {
+				return fmt.Errorf("open shard %s: %w", id, err)
```

Comment:
new option has _missing_ shard meaning. In general, missing isnt applicable when talking about shard configuration - it is a complex component, unlike, e.g., a single file/DB. Here we dont have 404 condition, so straighforward _uninited_ name fits better

Reply:
Rename all `missing` to `uninited`.